src/enable_variable_ops/ngraph_assign_op.cc

```diff
@@ -0,0 +1,189 @@
+/*******************************************************************************
+ * Copyright 2017-2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "tensorflow/core/common_runtime/dma_helper.h"
+#include "tensorflow/core/framework/op.h"
+#include "tensorflow/core/framework/op_kernel.h"
+#include "tensorflow/core/framework/resource_mgr.h"
+#include "tensorflow/core/lib/strings/strcat.h"
+
+#include "tensorflow/core/framework/op_kernel.h"
+#include "tensorflow/core/framework/tensor_types.h"
+#include "tensorflow/core/platform/default/logging.h"
+
+#include "ngraph/runtime/backend.hpp"
+#include "ngraph_catalog.h"
+#include "ngraph_freshness_tracker.h"
+#include "ngraph_timer.h"
+#include "ngraph_utils.h"
+#include "ngraph_var.h"
+
+using namespace std;
+namespace ng = ngraph;
+
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
+/* -------------------------------------------------
+//
+// NGraphAssignOp
+//
+---------------------------------------------------*/
+
+// Computes *input[0] = input[1]
+class NGraphAssignOp : public OpKernel {
+ private:
+  bool just_looking_;
+  bool copy_to_tf_;
+  int ng_graph_id_;
+  static int s_instance_count;
+  int my_instance_id{0};
+
+  // TODO(malikshr): Do we need these attributes, exist in TF Assign ops
+  // use_exclusive_lock_, validate_shape_, relax_constraints_;
+
+ public:
+  explicit NGraphAssignOp(OpKernelConstruction* context)
+      : OpKernel(context), just_looking_(false), copy_to_tf_(false) {
+    OP_REQUIRES_OK(context, context->GetAttr("just_looking", &just_looking_));
+    OP_REQUIRES_OK(context, context->GetAttr("copy_to_tf", &copy_to_tf_));
+    OP_REQUIRES_OK(context, context->GetAttr("ngraph_graph_id", &ng_graph_id_));
+
+    NGRAPH_VLOG(4) << "NGraphAssign:: Constructor called for: " << def().name()
+                   << " ,just looking " << just_looking_ << " ,copy-to-tf "
```

Comment:
```suggestion
                   << ", just looking " << just_looking_ << ", copy-to-tf "
```

Minor comma alignment in the logs. Applicable to other logs as well